### PR TITLE
Fix filtering out external features.

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
    Unreleased section, uncommenting the header as necessary.
 -->
 
-<!--## Unreleased-->
+## Unreleased
+
+* Fix an issue where package-external features could be included in queries that explicitly asked for them not to be. Specifically imports like `../paper-button/paper-button.html` weren't considering the resolved path like `bower_components/paper-button/paper-button.html` when determining externality.
```
Comment (FredKSchott) [inline CHANGELOG.md]: not major, but a little confused by what you mean that "imports wren't considering their own resolved path". Shouldn't this be that the analyzer isn't considering resolvedness?

File: src/model/document.ts [hunk 2]
```diff
@@ -335,7 +335,8 @@ export class Document implements Feature, Queryable {
       }
       if (feature.kinds.has('import') && options.imported) {
         const imprt = feature as Import;
-        if (options.externalPackages || !Package.isExternal(imprt.url)) {
+        if (options.externalPackages ||
```
Comment (FredKSchott) [inline src/model/document.ts]: can you add parens here? They make it so much easier to read/maintain when there are multiple logical operators in a conditional.

Comment (FredKSchott) [inline src/model/document.ts]: you could also pull `imprt.document && !Package.isExternal(imprt.document.url)` into a variable `isExternalDocument`

File: CHANGELOG.md (hunk 1 above)
Comment (rictic) [inline CHANGELOG.md]: Yeah, this wasn't clear. Rephrased to talk about user-visible concepts like "path in file" and "path on disk"

**Pull request merged**